lib/BUILD.bazel

```diff
@@ -1,3 +1,8 @@
+sh_test(
```

Comment:
This may need the copyright as well, but I'm not so sure with BUILD.bazel files...

Reply:
I checked in k/k and most (all?) of the bazel files didn't have a copyright header.